bims/views/map.py

```diff
@@ -46,4 +47,8 @@ def get_context_data(self, **kwargs):
             context['date_filter']['min'] = date_min.year
         if date_max:
             context['date_filter']['max'] = date_max.year
+        try:
```

Comment:
please swicth html in here between healthyrivers-map.html and map.html

Reply:
done